.github/workflows/rosa-cluster-auto-provision-on-schedule.yml

```
@@ -0,0 +1,36 @@
+name: ROSA Scheduled Create
+
+on:
+  schedule:
+    - cron: '0 7 * * *' # Runs every day at 7 AM UTC
+
+env:
+  OPENSHIFT_VERSION: 4.12.17
+
+jobs:
+
+  checkout:
+    name: Create ROSA cluster
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup ROSA CLI
+        uses: ./.github/actions/rosa-cli-setup
+        with:
+          aws-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-default-region: ${{ vars.AWS_DEFAULT_REGION }}
+          rosa-token: ${{ secrets.ROSA_TOKEN }}
+
+      - name: Create ROSA Cluster
+        run: ./rosa_create_cluster.sh
+        working-directory: provision/aws
+        env:
+          VERSION: ${{ env.OPENSHIFT_VERSION }}
+          CLUSTER_NAME: shared-crossdc
+          REGION: ${{ vars.AWS_DEFAULT_REGION }}
+          COMPUTE_MACHINE_TYPE: m5.xlarge
+          MULTI_AZ: false
+          REPLICAS: 2
```

Instead of repeating steps from `rosa-cluster-create.yml` it would be better to just run that workflow like this:

```suggestion
      - run: gh workflow run -R keycloak/keycloak-benchmark rosa-cluster-create.yml
        if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak-benchmark'
```

This prevents us from having to maintain two workflows with the same steps. Alternatively the `rosa-cluster-create.yml` can be converted to a composite workflow (like we do in keycloak-rel), but that's a bit messier IMO.

---

ok, I was not sure, if the `workflow_dispatch` type workflows can be reused ? @stianst that would make sense if we can run it like a downstream workflow to re-use the `rosa-cluster-create.yml`

---

applied the changes